Terraform_modules/rds_serverless_cluster/variables.tf

```diff
@@ -36,14 +41,21 @@ variable "database_subnet_ids" {
   description = "This is a list of subnet ids that the database cluster will be created across. The minimum number of subnets that can be supplied is 2."
 }
 
-variable "database_availability_zone" {
-  type        = string
-  description = "This is the availability zone that the database instance will be created on."
+variable "replicate_on_availability_zones" {
```

Comment:
Could you add a check that the primary AZ is not in this list? (Assuming that's the desired behaviour)

Also, you've got a comment below about "optional variables" - looks like this variable should be in that section

Reply:
That was in a previous commit, I removed that because your not allowed to reference other variables within a conditional check.